spec/system/admin/products_spec.rb

```diff
@@ -109,7 +109,7 @@
       expect(product.primary_taxon_id).to eq(taxon.id)
       expect(product.variants.first.price.to_s).to eq('19.99')
       expect(product.on_hand).to eq(5)
-      expect(product.tax_category_id).to eq(tax_category.id)
+      expect(product.variants.first.tax_category_id).to eq(tax_category.id)
```

Comment:
I'm a bit concerned about this `variants.first`: this give a special behavior to the first created (I suppose?) variant of a product. I can't imagine another solution, but wanted to mention this.

Reply:
The test is creating a new product via the UI and when that happens a new variant is automatically added to the new product. I'm using `product.variants.first` here because we don't have a reference to the variant before the test starts. There's only one variant created, so grabbing the first one is the simplest way, I'm not sure how else we could do it. This checks that the tax category the user selected in the new product form is correctly saved on the variant object.